packages/data/README.md

```diff
@@ -98,9 +98,11 @@ registerStore( 'my-shop', {
 		},
 	},
 } );
+
+register( store );
 ```
 
-The return value of `registerStore` is a [Redux-like store object](https://redux.js.org/basics/store) with the following methods:
+The return value of `createReduxStore` is a [Redux-like store object](https://redux.js.org/basics/store) with the following methods:
```

Comment:
Is is still true?

Reply:
no, you only get `{ name, instantiate }`

Reply:
It should be now better with 5f3a817.